src/applications/facility-locator/components/SearchControls.jsx

```diff
@@ -55,22 +70,46 @@ class SearchControls extends Component {
     this.props.onSubmit();
   };
 
+  hideClearInput = () => {
+    this.inputClearRef.current.style.display = 'none';
+    this.inputRef.current.value = '';
+    focusElement('#street-city-state-zip');
+  };
+
+  renderClearInput = () => {
+    if (window.Cypress || !environment.isProduction()) {
+      return (
+        <i
+          aria-hidden="true"
+          className="fas fa-times-circle fa-clear"
+          id="clear-input"
+          ref={this.inputClearRef}
+          onClick={this.hideClearInput}
```

Comment:
The "React way" to do this would be to dispatch an action that clears the `currentQuery.searchString`, then let that change propagate to the UI.